### PR TITLE
SFB-118: translations

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## ID
 [SFB-118](https://binnenland.atlassian.net/browse/SFB-118)

 ## Description

In the form builder a mix of dutch and english is used atm. Change everything to the ~english~ Dutch translation. As we are already translating i added a translation package to the project so we can find almost all of the translations in one place.

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [x] Breaking change
 - [ ] Other

 ## How to test

 1. See if everything is translated to Dutch
 2. Some things could not be translated because of it is set in a component (form overview table)

## Todo

- [ ] translate the ttl values

 ## Links to other PR's

 - /

### CODE REVIEW

File: app/formats.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export default {
+  time: {
+    hhmmss: {
```
Comment (JonasVanHoof) [inline app/formats.js:3]: This is generated when setting up the translations

File: app/routes/application.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 import Route from '@ember/routing/route';
 import { action } from '@ember/object';
+import { service } from '@ember/service';
 
 export default class FormsPlaygroundRoute extends Route {
+  @service intl; // Inject the intl service
+
+  beforeModel() {
+    // Set the locale before the model is loaded (you can adjust the locale as needed)
+    this.intl.setLocale(['NL']);
```
Comment (JonasVanHoof) [inline app/routes/application.js:10]: Language code for dutch is `NL`

File: app/templates/codelijsten.hbs [hunk 3]
```diff
@@ -64,13 +67,13 @@
...
                   <strong>form:options</strong>
-                  en vervang de link achter
+                  and replace the link behind
                   <strong>conceptScheme</strong>
-                  om de concept scheme te activeren.</li>
-                <li>Sommige links omvatten meer informatie over de concept
-                  scheme zelf. Klik op de link om die informatie te achterhalen.</li>
+                  to activate the concept scheme</li>
+                <li>Some of the links contain more information about the concept
+                  scheme itself. Click the link top get the information</li>
```
Comment (JonasVanHoof) [inline app/templates/codelijsten.hbs:76]: Didn't put this in the translation file yet. As it has some styling applied to it. Plus possibly this page will be replaced with the new code lists view from #84

File: app/templates/index.hbs [hunk 4]
```diff
@@ -31,6 +31,9 @@
           <c.header>
             <AuDataTableThSortable @label='Naam' @field='label' @currentSorting={{this.sort}}
               class="au-u-visible-medium-up" />
+            <AuDataTableThSortable @label='Beschrijving' @field='comment' @currentSorting={{this.sort}}
+              class="au-u-visible-medium-up" />
+
             <AuDataTableThSortable @label='Aangemaakt op' @field='created' @currentSorting={{this.sort}}
               class="au-u-visible-small-up" />
             <AuDataTableThSortable @label='Aangepast op' @field='modified' @currentSorting={{this.sort}}
```
Comment (JonasVanHoof) [inline app/templates/index.hbs:39]: This is an component that cannot read the translations. We could change it to a normal table as we did for the codelists

**Review by ZijneDwaasheid — COMMENTED** [review]
looks good. Like I told you it feels a bit strange to see all these things in Dutch now, but it is what it is. Next up, form.tll translations :)

**Comment by JonasVanHoof** [comment]
> looks good. Like I told you it feels a bit strange to see all these things in Dutch now, but it is what it is. Next up, form.tll translations :)

Yes thought about that! And actually already changed it but than thought that is not a good idea as this will make the builder show dutch and english combined. Haha the whole point of this

**Pull request merged**